### PR TITLE
add: `p3x-onenote-app`

### PR DESCRIPTION
**Pull request by harrietobrien**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/p3x-onenote-app/p3x-onenote-app.pacscript [hunk 1]
```diff
@@ -0,0 +1,25 @@
+pkgname="p3x-onenote-app"
+_name="P3X-OneNote"
+gives="p3x-onenote"
+pkgver="2024.10.117"
+url="https://www.corifeus.com/onenote"
+pkgdesc="P3X OneNote Linux is an independent browser window for OneNote online"
+noextract=("${_name}-${pkgver}.Appimage")
+depends=()
```
Comment (vigress8) [inline packages/p3x-onenote-app/p3x-onenote-app.pacscript:8]:
```suggestion
```
AppImages aren't auto-extracted anyway

File: packages/p3x-onenote-app/p3x-onenote-app.pacscript [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "p3x-onenote.desktop"
+  "onenote-svg.html"
+)
+sha256sums="f95c4dc529f86b00a15997ec2024a2ca1a139048d2cf816dfd6ade721d5d635e"
+arch=("amd64")
+maintainer="harrietobrien <harrietobrien@protonmail.com>"
+
+package() {
+  cd "${srcdir}"
+  chmod +x "${_name}-${pkgver}.AppImage"
```
Comment (vigress8) [inline packages/p3x-onenote-app/p3x-onenote-app.pacscript:20]:
```suggestion
```
Redundant lines

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/p3x-onenote-app/onenote-svg.html
[the remark this answers is not in the document]
Comment (harrietobrien) [reply packages/p3x-onenote-app/onenote-svg.html:1]: I can add link to sources and install that way later (at work rn)

[the remark this answers is not in the document]
Comment (vigress8) [reply packages/p3x-onenote-app/onenote-svg.html:1]: No, I meant - why is the file called `onenote-svg.html` is it's an svg? Why not call it `onenote.svg`

File: packages/p3x-onenote-app/onenote.svg [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<svg version="1.1" id="Livello_1" xmlns:x="http://ns.adobe.com/Extensibility/1.0/" xmlns:i="http://ns.adobe.com/AdobeIllustrator/10.0/" xmlns:graph="http://ns.adobe.com/Graphs/1.0/" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 2744.833 2553.333" enable-background="new 0 0 2744.833 2553.333" xml:space="preserve">
```
Comment (Elsie19) [inline packages/p3x-onenote-app/onenote.svg:1]: Can you move this file elsewhere. I know it's an SVG but I don't like adding images to the repo.

**Review by Elsie19 — DISMISSED** [review]

File: packages/p3x-onenote-app/p3x-onenote-app.pacscript [hunk 4]
```diff
@@ -0,0 +1,25 @@
+pkgname="p3x-onenote-app"
+_name="P3X-OneNote"
+gives="p3x-onenote"
+pkgver="2024.10.117"
+url="https://www.corifeus.com/onenote"
+pkgdesc="Independent browser window for OneNote online"
+depends=(
+  "perl-base"
+  "liblocale-gettext-perl"
+)
```
Comment (vigress8) [inline packages/p3x-onenote-app/p3x-onenote-app.pacscript:10]:
```suggestion
)
incompatible=('debian:sid')
```

**Review by vigress8 — APPROVED** [review]

**Pull request merged**